libs/common/src/lib/tasks-automation/task-forms/ask/ask.component.html

```diff
@@ -0,0 +1,56 @@
+<div class="page-spacing">
+  <header>
+    <nsi-back-button [link]="backRoute | async">
+      {{ 'tasks-automation.bach-to-tasks' | translate }}
+    </nsi-back-button>
+
+    <div>
+      @if (task) {
+        <h1 class="page-title">{{ task.parameters.name }}</h1>
+      } @else {
+        <h1 class="page-title">{{ 'tasks-automation.generator.title' | translate }}</h1>
+        <p class="page-description">{{ 'tasks-automation.generator.description' | translate }}</p>
+      }
+    </div>
+  </header>
+
+  <app-task-form
+    triggerDescription="tasks-automation.generator.trigger-description"
+    activateButtonLabel="tasks-automation.generator.footer.activate-button-label"
+    footerNoteAutomation="tasks-automation.generator.footer.apply-automation"
+    footerNoteOneTime="tasks-automation.generator.footer.apply-one-time"
+    [validFormInside]="askForm.valid"
+    (cancel)="backToTaskList()"
+    (activate)="activateTask($event)">
```

Comment:
a minor thing, in the labeler task we added `[task]="task"`, so that existing tasks can be viewed.